app/src/main/res/layout/item_comment_product.xml

```diff
@@ -12,27 +13,29 @@
     <android.support.v7.widget.CardView
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_margin="@dimen/dp_3"
-        android:padding="@dimen/dp_3"
+        android:layout_margin="@dimen/dp_1"
+        android:padding="@dimen/dp_2"
         >
         <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_margin="@dimen/dp_5"
-            android:layout_marginTop="@dimen/dp_10"
             android:orientation="vertical"
+            android:padding="@dimen/dp_2"
+            android:paddingEnd="@dimen/dp_2"
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 14) you should *also* add `android:paddingRight="@dimen/dp_2"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.